packages/cli/contracts/src/prototypes/Tables23.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+/* Autogenerated file. Do not edit manually. */
+
+import { Table2, Table2Data } from "./../tables/Table2.sol";
+import { Table3 } from "./../tables/Table3.sol";
+import { Enum1 } from "./../Types.sol";
+
+library Tables23 {
+  function create(bytes32 key, Table2Data memory _data_Table2, Enum1 _value_Table3) internal {
```

Comment:
What do you think about generating these without the `_data` or `_value` prefix? Thinking about UX of using the libraries, especially with a prototypes with lots of value I see people using names arguments like

```solidity
Tables23.create({
  _key: 0x01,
  _data_Table2: table2Data,
  _value_Table3: enum1
});
```

and that would be nicer like

```solidity
Tables23.create({
  _key: 0x01,
  _Table2: table2Data,
  _Table3: enum1
});
```

Reply:
updated it to only resort to field name prefixes for multi-argument tables